website/partials/status.html

```diff
@@ -0,0 +1,40 @@
+<div class="lh-1-5">
+  <h1 class="mb-50">Project Status</h1>
+
+  <h2>Is Protractor actively maintained? {{isEOL}} {{isEOCS}}</h2>
+  <div ng-hide="isEOL">
+    <p>
+      Although active development on Protractor ended in December of 2022, the project will continue to be supported under <strong>Long Term Support</strong> and will continue to receive critical security &amp; compatibility fixes via the community through August 2023. This is important as there are many actively developed projects still benefiting from their existing code coverage, and we anticipate Protractor to continue functioning well into the future. 
+    </p>
+    <p>
+      In short: Protractor's current status is <strong>Long Term Support</strong> and <strong>will End Of Life in September 2023</strong>.
+    </p>
+  </div>
+  
+  <div ng-show="isEOL">
+    <p>
+      Although active development on Protractor ended in December of 2022, the project continued to be supported under <strong>Long Term Support</strong> with critical security &amp; compatibility fixes via the community through August 2023.
+    </p>
+    <p>
+      As of September 2023, Protractor's status is officially <strong>End Of Life</strong>.
+    </p>
+  </div>
+  
+  <h2>What does End Of Life mean?</h2>
+  The packages will remain accessible on GitHub, npm, Bower, and Release archive. This website will also remain here indefinitely. The project will be archived on GitHub and remain accessible indefinitely, but will no longer be open to issues / comments / etc. 
```

Comment:
> This website will also remain here indefinitely.

@mgechev, are we comfortable committing to the website staying up _forever_? I doubt GitHub Pages would push a significant breaking change here, but if they did and it was a non-trivial effort to move off it, I expect we probably wouldn't bother given Internet Archive as the ultimate workaround.

Reply:
Yeah, I agree. Let's change to "foreseeable future"

We're not anticipating making the website unavailable, but inevitably it'll happen one day.